components/brave_rewards/core/common/request_signer_unittest.cc

```diff
@@ -28,8 +28,7 @@ class RewardsRequestSignerTest : public testing::Test {
 
 TEST_F(RewardsRequestSignerTest, GetDigest) {
   auto seed = Signer::GenerateRecoverySeed();
-  auto digest =
-      RequestSigner::GetDigest(base::as_bytes(base::make_span("hello world")));
+  auto digest = RequestSigner::GetDigest(base::as_byte_span("hello world"));
   EXPECT_EQ(digest, "SHA-256=QwZGhH5wNEwJ9Yc56Z1byW6sjV/nKVzxlrmGJ5h2v5s=");
 }
```

Comment:
It looks it like it should be `byte_span_from_cstring` which would change what this test tests but would be more correct.

Reply:
Fixed. Thanks. Using `byte_span_with_nul_from_cstring` to preserve the `nul` as in the example above.